cmd/acra_configui/acra_configui.go

```diff
@@ -105,12 +116,37 @@ func SubmitSettings(w http.ResponseWriter, r *http.Request) {
 	w.Write(jsonToServer)
 }
 
+func parseTemplate() {
+	tplPath := fmt.Sprintf(filepath.Join(*staticPath, "index.html"))
+	tplPath, err = utils.AbsPath(tplPath)
+	if err != nil {
+		log.WithError(err).Errorf("no config file[%v]", tplPath)
+	}
+
+	parsedTemplate, err = template.ParseFiles(tplPath)
+	if err != nil {
+		log.WithError(err).Errorf("err while parsing template - %v", tplPath)
+		syscall.Exit(1)
```

Comment:
what about to return error object and handle it on up level? then we can easily test it if need. otherwise it is non-testable function

Reply:
done